windows.md

```diff
@@ -397,7 +397,59 @@ With those compatibility things out of the way, you're ready to start the system
     ```
 
     It should print out a welcome message like this:<br><br>
-    <img src="macos-6-docker-hello-world.png"><br><br>
+    <img src="macos-7-docker-hello-world.png"><br><br>
+
+    If opening Docker Desktop or running the command above shows a message that says `Docker Desktop - Windows HyperVisor is not Present`, you may need to enable virtualization on your machine.
```

Comment:
1. let's make sure that we start this as early as the first error could occur (check if there is any step before this where it would be better to start this guide)
2. check if we should break this up into a separate optional step, to make it easier for people to skip this section

Reply:
I moved it after the Docker Desktop installation because the error would happen when they try to open Docker Desktop after the installation

https://github.com/upleveled/system-setup/pull/16/commits/d98cc2a398beaeea06481e1f957032a6bbd27d81

I was thinking that they can already skip after reading this first line

> If opening Docker Desktop from **Option A or B** shows an error, you may need to enable virtualization on your machine